Namespace(cuda=True, directory='program_data/cpp_babi_format_Sep-29-2018-10-ce3766a', is_training_ggnn=True, log_path='program_data/cpp_babi_format_Sep-29-2018-10-ce3766a/logs', lr=0.01, manualSeed=None, model_path='program_data/cpp_babi_format_Sep-29-2018-10-ce3766a/cpp_babi_format_Sep-29-2018-10-ce3766a-10.cpkl', n_classes=10, n_hidden=50, n_steps=10, niter=2, size_vocabulary=55, state_dim=5, test_batch_size=32, testing=False, train_batch_size=32, training=True, training_percentage=1.0, verbal=True, workers=2)
Random Seed:  2123
  0% 0/10 [00:00<?, ?it/s] 30% 3/10 [00:00<00:00, 29.54it/s] 60% 6/10 [00:00<00:00, 28.43it/s] 90% 9/10 [00:00<00:00, 26.51it/s]100% 10/10 [00:00<00:00, 26.25it/s]
Number of all training data : 3330
Max node id : 48
  0% 0/10 [00:00<?, ?it/s] 40% 4/10 [00:00<00:00, 37.07it/s]100% 10/10 [00:00<00:00, 38.01it/s]
Number of all testing data : 1670
Max node id : 46
/opt/conda/lib/python3.6/site-packages/torch/onnx/utils.py:365: UserWarning: ONNX export failed on ATen operator stack because torch.onnx.symbolic.stack does not exist
  .format(op_name, op_name))
/opt/conda/lib/python3.6/site-packages/tensorboardX/pytorch_graph.py:43: UserWarning: Error getting attributes of node %98 : Dynamic = onnx::Constant[value={1}](), scope: GGNN/Propogator[propogator], error is VariableType::ID() not implemented
  warnings.warn("Error getting attributes of node {}, error is {}".format(attrs, e))
/opt/conda/lib/python3.6/site-packages/tensorboardX/pytorch_graph.py:43: UserWarning: Error getting attributes of node %161 : Dynamic = onnx::Constant[value={1}](), scope: GGNN/Propogator[propogator], error is VariableType::ID() not implemented
  warnings.warn("Error getting attributes of node {}, error is {}".format(attrs, e))
/opt/conda/lib/python3.6/site-packages/tensorboardX/pytorch_graph.py:43: UserWarning: Error getting attributes of node %224 : Dynamic = onnx::Constant[value={1}](), scope: GGNN/Propogator[propogator], error is VariableType::ID() not implemented
  warnings.warn("Error getting attributes of node {}, error is {}".format(attrs, e))
/opt/conda/lib/python3.6/site-packages/tensorboardX/pytorch_graph.py:43: UserWarning: Error getting attributes of node %287 : Dynamic = onnx::Constant[value={1}](), scope: GGNN/Propogator[propogator], error is VariableType::ID() not implemented
  warnings.warn("Error getting attributes of node {}, error is {}".format(attrs, e))
/opt/conda/lib/python3.6/site-packages/tensorboardX/pytorch_graph.py:43: UserWarning: Error getting attributes of node %350 : Dynamic = onnx::Constant[value={1}](), scope: GGNN/Propogator[propogator], error is VariableType::ID() not implemented
  warnings.warn("Error getting attributes of node {}, error is {}".format(attrs, e))
/opt/conda/lib/python3.6/site-packages/tensorboardX/pytorch_graph.py:43: UserWarning: Error getting attributes of node %413 : Dynamic = onnx::Constant[value={1}](), scope: GGNN/Propogator[propogator], error is VariableType::ID() not implemented
  warnings.warn("Error getting attributes of node {}, error is {}".format(attrs, e))
/opt/conda/lib/python3.6/site-packages/tensorboardX/pytorch_graph.py:43: UserWarning: Error getting attributes of node %476 : Dynamic = onnx::Constant[value={1}](), scope: GGNN/Propogator[propogator], error is VariableType::ID() not implemented
  warnings.warn("Error getting attributes of node {}, error is {}".format(attrs, e))
/opt/conda/lib/python3.6/site-packages/tensorboardX/pytorch_graph.py:43: UserWarning: Error getting attributes of node %539 : Dynamic = onnx::Constant[value={1}](), scope: GGNN/Propogator[propogator], error is VariableType::ID() not implemented
  warnings.warn("Error getting attributes of node {}, error is {}".format(attrs, e))
/opt/conda/lib/python3.6/site-packages/tensorboardX/pytorch_graph.py:43: UserWarning: Error getting attributes of node %602 : Dynamic = onnx::Constant[value={1}](), scope: GGNN/Propogator[propogator], error is VariableType::ID() not implemented
  warnings.warn("Error getting attributes of node {}, error is {}".format(attrs, e))
/opt/conda/lib/python3.6/site-packages/tensorboardX/pytorch_graph.py:43: UserWarning: Error getting attributes of node %665 : Dynamic = onnx::Constant[value={1}](), scope: GGNN/Propogator[propogator], error is VariableType::ID() not implemented
  warnings.warn("Error getting attributes of node {}, error is {}".format(attrs, e))
[0/2][0/105] Loss: 2.3325
[0/2][11/105] Loss: 2.2778
[0/2][22/105] Loss: 2.2233
[0/2][33/105] Loss: 2.0974
[0/2][44/105] Loss: 1.9706
[0/2][55/105] Loss: 1.9839
[0/2][66/105] Loss: 2.0393
[0/2][77/105] Loss: 1.9195
[0/2][88/105] Loss: 1.8823
[0/2][99/105] Loss: 1.8876
Test set: Average loss: 0.0598, Accuracy: 968/1670 (57%)
[1/2][0/105] Loss: 1.8755
[1/2][11/105] Loss: 1.9336
[1/2][22/105] Loss: 1.8099
[1/2][33/105] Loss: 1.8856
[1/2][44/105] Loss: 1.8763
[1/2][55/105] Loss: 1.8292
[1/2][66/105] Loss: 1.8705
[1/2][77/105] Loss: 1.7068
[1/2][88/105] Loss: 1.8249
[1/2][99/105] Loss: 1.8221
Test set: Average loss: 0.0576, Accuracy: 1089/1670 (65%)
(57%)
(65%)
